lib/internal/bootstrap/node.js

```diff
@@ -311,6 +292,36 @@
     }
   }
 
+  function setupRepl() {
+    // used by https://github.com/nodejs/repl
+    if (process.env.NODE_EXTERNAL_REPL_MODULE) {
+      const CJSModule = NativeModule
+        .require('internal/modules/cjs/loader');
+      CJSModule._load(process.env.NODE_EXTERNAL_REPL_MODULE, undefined, true);
```

Comment:
Try setting the `isMain` param from `true` to `false` 👆 _(if it works then roll with it)_

Reply:
i assumed we would want it to be true, since it's the main entry in this case?